Makefile

```
@@ -6,6 +6,7 @@ SOURCE   = $(URL_BASE)/$(NAME)-$(VERSION).tar.$(SRC_EXT)
 ID := $(shell . /etc/os-release; echo $$ID)
 # Suse has to be different, only care about CentOS or SUSE
 ifneq ($(ID),centos)
+SPEC_UPDATE := $(shell cp slurm.spec_sles12 slurm.spec)
```

I believe you simplify this technique into:

```suggestion
$(shell cp slurm.spec_sles12 slurm.spec)
```